test/test/assets/hls-ts-h265/index.m3u8

```diff
@@ -4,4 +4,6 @@
 #EXT-X-MEDIA-SEQUENCE:0
 #EXTINF:12.000000,
 0.ts
+#EXTINF:12.000000,
+0.ts
```

Comment:
Just want to confirm that you intended to put 0.ts, not 1.ts like in other places.

Reply:
Yes.  I didn't want to create a new segment and store it in the repo.  In sequence mode, the timestamps don't matter, so I can just repeat the first segment again to make the content longer.